hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/audit/parser/common/DatabaseHelper.java

```diff
@@ -213,27 +223,24 @@ public static String executeTemplate(String dbName, String template)
   private static String executeStatement(String dbName, String sql)
       throws SQLException {
     StringBuilder result = new StringBuilder();
-    ResultSet rs = null;
-    Statement st = null;
-    ResultSetMetaData rsm = null;
-    try(Connection connection = getConnection(dbName)) {
-      //loadProperties();
-
-      if(connection != null){
-        st = connection.createStatement();
-        rs = st.executeQuery(sql);
-        if(rs != null) {
-          rsm = rs.getMetaData();
-          int cols = rsm.getColumnCount();
-          while(rs.next()){
-            for(int index =1; index<=cols; index++){
-              result.append(rs.getObject(index) + "\t");
+    ResultSetMetaData rsm;
+    try (Connection connection = getConnection(dbName)) {
+      if (connection != null){
+        try (Statement st = connection.createStatement()) {
+          try (ResultSet rs = st.executeQuery(sql)) {
```

Comment:
These can be collapsed to a single `try` (last 2 currently, all 3 after changing `getConnection`).

Reply:
But that would result in Sonarlint warning.

Reply:
Ahh, ok I see what you mean, nm!